src/nrniv/netpar.cpp

```diff
@@ -124,18 +124,10 @@ double ncs_netcon_localmindelay( int srcgid )
 //get the number of netcons for an object, if it sends here
 int ncs_netcon_count( int srcgid, bool localNetCons )
 {
-    PreSyn *ps{nullptr};
-    if( localNetCons ) {
-        auto iter = gid2out_.find(srcgid);
-        if (iter != gid2out_.end()) {
-            ps = iter->second;
-        }
-    }else{
-        auto iter = gid2in_.find(srcgid);
-        if (iter != gid2in_.end()) {
-            ps = iter->second;
-        }
-    }
+    const auto& map = localNetCons ? gid2out_ : gid2in_;
+    const auto iter = map.find(srcgid);
+    PreSyn* ps{iter != map.end ? iter->second : nullptr};
```

Comment:
```suggestion
    PreSyn* ps{iter != map.end() ? iter->second : nullptr};
```
I guess this isn't compiled 🤔